.github/workflows/wheels.yml

```diff
@@ -93,12 +96,24 @@ jobs:
             echo "CIBW_ENVIRONMENT_MACOS=$CIBW PKG_CONFIG_PATH=$PKG_CONFIG_PATH DYLD_LIBRARY_PATH=$DYLD" >> "$GITHUB_ENV"
           fi
 
+      - name: Setup iOS
+        if: matrix.buildplat[0] == 'ios'
+        run: |
+          if [[ "${{ matrix.buildplat[3] }}" == 'accelerate' ]]; then
+            # Accelerate with ILP64 requires iOS 16.4 or higher
+            CIBW="IPHONEOS_DEPLOYMENT_TARGET=17.0 INSTALL_OPENBLAS=false RUNNER_OS=iOS"
```

Comment:
iOS 17 is only 2 years old, so this means a lot of apps, including the default Briefcase configuration, will end up using the unaccelerated build, which may be ["100x slower"](https://github.com/numpy/numpy/blob/v2.3.5/numpy/meson.build#L185-L187). 

Could we avoid this by building in non-ILP64 mode? As I understand it, the only advantage of ILP64 is that it allows arrays with more than 2 billion elements, which, on a phone, is far less likely to be a problem than a 100x slowdown. In that case, there probably isn't even any need for a separate iOS 17.0 build, and the workflows can all be simplified.

Reply:
It may be "only" 2 years old, but a feature of the iOS ecosystem is by mass and near-immediate adoption of OS updates. Based on [this chart](https://telemetrydeck.com/survey/apple/iOS/majorSystemVersions/), as of today, 97.5% of devices are on iOS 17 or newer. I included the "non accelerate" wheels mostly as a convenience so that a a default iOS install will have access to *something*.

I can take a look to see if it's possible to pick up the older accelerate implementation on iOS13+.